primitives/core/src/offchain/mod.rs

```diff
@@ -260,26 +260,22 @@ impl Timestamp {
 bitflags::bitflags! {
 	/// Execution context extra capabilities.
 	pub struct Capabilities: u32 {
-		/// Access to transaction pool.
-		const TRANSACTION_POOL = 0b0000_0000_0001;
 		/// External http calls.
-		const HTTP = 0b0000_0000_0010;
+		const HTTP = 0b0000_0000_0001;
```

Comment:
(Purely stylistic personal opinion/suggestion, feel free to ignore if you prefer it this way.)

For defining flags I often like to use bitshifts, e.g.:

```rust
const HTTP = 1 << 0;
const KEYSTORE = 1 << 1;
const RANDOMNESS = 1 << 2;
```

This immediately tells you which bit it is, and especially helps when you have a lot of flags.